lib/travis/build/script/shared/rvm.rb

```diff
@@ -186,6 +190,18 @@ def setup_rvm_aliases
           def without_teeny?(version)
             version =~ /\A(\d+)(\.\d+)\z/
           end
+
+          def install_bundler
+            sh.if "! $(command -v bundler)" do
+              sh.echo "Installing Bundler", ansi: :yellow
```

Comment:
Could this be wrapped in a fold? And maybe also a newline before the fold?
I'd say the name of the fold should just be the command, the header `Installing Bundler` seems unnecessary when compared to `rvm use` line.